utils.py

```diff
@@ -27,6 +28,10 @@ def getCVE_Dict(year: str) -> {}:
     :return: CVE dictionary
     """
     cve_dict = {}
+    if 'nvd' not in listdir():
+        pathlib.Path('nvd').mkdir(parents=True, exist_ok=True)
```

Comment:
InstalledSoftware: separate this file into 2 classes - one that queries local registry, one that returns installed programs